src/Components/DiaryHome.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button } from '@material-ui/core';
+
+
+const useStyle = makeStyles((themes) => ({
+
+    root: {
+        width: "50%",
+        padding: "10px",
+        textAlign: "center",
+        backgroundColor: "#0d47a1",
+
+        [themes.breakpoints.down('xs')]: {
+            width: "100%"
+        }
+    },
+    title: {
+        backgroundColor: "#bbdefb",
+        borderRadius: "100px",
+        border: "none",
+        height: "20px",
+        width: "70%",
+        margin: "20px",
+        padding: "10px",
+        paddingLeft: "20px",
+    },
+
+    discription: {
+        backgroundColor: "#bbdefb",
+        borderRadius: "20px",
+        border: "none",
+        width: "70%",
+        margin: "20px",
+        padding: "10px",
+        paddingLeft: "20px",
+    },
+    submit: {
+        backgroundColor: "#bbdefb",
+        width: "70%",
+        borderRadius: "100px",
+        margin: "10px"
+    }
+}))
+
+function DiaryHome() {
+
+    const classes = useStyle();
+
+    const [title, setTitle] = useState("");
+    const [description, setDescription] = useState("");
+
+    //Read title input
+    function titleHandle(e) {
+        setTitle(e.target.value)
+    }
+    //Read description input
+    function descriptionHandle(e) {
+        setDescription(e.target.value);
+    }
```

Comment:
you can use an arrow function for each onChange like `onChange={(e)=>setTittle(e.target.value)}`